src/bonsai/Makefile

```diff
@@ -199,14 +199,14 @@ endif
 	cd build && . env/$(VENV_ACTIVATE) && $(PIP) download flask $(PYPI_PLATFORM) --python-version $(PYPI_VERSION) --implementation $(PYPI_IMP) --only-binary=:all: --dest=./wheels
 
 	# jQuery is required for web UI functionality
-	cd build/bonsai/bim/data/webui/static/js/ && wget https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js
+	cd build/bonsai/bim/data/webui/static/js/ && wget --tries=5 --timeout=60 --waitretry=5 --retry-connrefused --continue https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js
```

Comment:
Since this is largely repetetive I suggest extracting it into a function, for the benefit that functions give.

```make
Download=cd $1 && wget --tries=5 --timeout=60 --waitretry=5 --retry-connrefused --continue $2

...

	$(call Download, build/bonsai/bim/data/webui/static/js/,https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js)
	...
	$(call Download, build, -O ifc2json.zip https://github.com/IFCJSON-Team/IFC2JSON_python/archive/refs/heads/master.zip)
```

Haven't been doing raw gnumake for a while so syntax might be slightly off.